test/TestExtensions/TestablePluginCredentialProvider/TestableCredentialProvider.csproj

```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="6.0.4" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
Does this project not import the props that contain `NewtonsoftJsonVersionCore`?

Reply:
I think it does, but I have a feeling that some people copy this project to start creating their own credential provider. So, I tried to keep this project file independent so that it can be copied elsewhere and still work. Although I now noticed that there are imports to our common props and targets files, so I should change the version to use the property value.

Reply:
I think this is the old cred provider. So not super relevant for new projects.
Yeah, I would just use the property value.